specification/resources/resource-manager/Microsoft.Resources/stable/2016-06-01/subscriptions.json

```diff
@@ -323,6 +315,13 @@
     }
   },
   "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription."
+    },
     "ApiVersionParameter": {
       "name": "api-version",
       "in": "query",
```

Comment:
please, add `"x-ms-parameter-location": "client"` to the `ApiVersionParameter` as well.